src/static_canvas.class.js

```diff
@@ -232,7 +232,7 @@
      * @private
      */
     _isRetinaScaling: function() {
-      return (fabric.devicePixelRatio !== 1 && this.enableRetinaScaling);
+      return (fabric.devicePixelRatio > 1 && this.enableRetinaScaling);
```

Comment:
you also need to change getRetinaScaling to return a number that is at minimum 1 i think.

Reply:
Done

Reply:
Maybe make sense to switch places of these parts for performance? Checking a boolean variable should be faster than comparing two values. Like this: `this.enableRetinaScaling && fabric.devicePixelRatio > 1`.

Reply:
@Dok11 i don't sure. JS engine probably is don't know whats variable contain, so will not check variable as boolean directly. But symbol `>` may suggest that it's just math operation which is instant

Reply:
@vitonsky I even tested it, and it seems like it doesn't care :D
https://jsbench.me/zkkxq3rjv0/1

![image](https://user-images.githubusercontent.com/2697890/147568070-ac57d0e8-b287-4b00-98ec-dfc99c9f8578.png)

Reply:
i m sure performance is not an issue here, but in the JS bench seems like Math.random and toFixed are actually the 2 things that will matter for performance. no?